pkg/plugin/linuxutil/netstat_stats_linux.go

```diff
@@ -269,36 +245,12 @@ func (nr *NetstatReader) updateMetrics() {
 		metrics.TCPStateGauge.WithLabelValues(state).Set(float64(v))
 	}
 
-	for remoteAddr, v := range nr.connStats.TcpSockets.socketByRemoteAddr {
-		addrPort, err := netip.ParseAddrPort(remoteAddr)
-		if err != nil {
-			nr.l.Error("Failed to parse remote address", zap.Error(err))
-			continue
-		}
-		addr := addrPort.Addr().String()
-		port := strconv.Itoa(int(addrPort.Port()))
-		if !validateRemoteAddr(addr) {
```

Comment:
let's only count valid addresses? Check if it's an empty string?

Reply:
`ParseAddrPort ` will handle empty strings